### ISSUE TITLE
Error when try to build for production

### ISSUE THREAD

**Issue by NashIlli** [AnthonyNahas/ngx-linkifyjs#24]
### Bug Report or Feature Request (mark with an `x`)
```
- [X] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?
Ubuntu 18

### Versions
Angular CLI: 7.0.6
Node: 10.6.0
Ngx-linkifyjs 1.2.1

### Repro steps
I tried build for producction my Angular app and I got the following error
![image](https://user-images.githubusercontent.com/6954280/49012283-b7a08800-f157-11e8-8604-80a76b1062ba.png)

**Comment by AnthonyNahas** [comment]
which version are you using ? I've just fixed that yesterday... The version [1.2.1](https://www.npmjs.com/package/ngx-linkifyjs) is the latest release!

**Comment by NashIlli** [comment]
I'm using version 1.2.1. I've just updated the library!

**Comment by webcat12345** [comment]
@AnthonyNahas Having same error.
and version is "ngx-linkifyjs": "^1.2.1",

**Comment by AnthonyNahas** [comment]
Bug found! I will solve this issue and patch the library asap! Thanks for reporting 👍

**Comment by AnthonyNahas** [comment]
I've just patched [ngx-linkifyjs v1.2.2](https://www.npmjs.com/package/ngx-linkifyjs).

I would appreciate your feedback! Please feel free to reopen the issue if that problem doesn't disappear!

**Comment by NashIlli** [comment]
@AnthonyNahas it works! Thanks!
